src/abstracts/egamefactory.py

```diff
@@ -270,3 +270,102 @@ async def gather_players(self) -> int:
         self.logging.info(f"number of players {self._num_players}")
 
         return self._num_players
+
+    async def check_continue(self, min_players: int = 1):
+        """
+        Brings up a dialogue on the main channel to ask players whether they want to continue with
+        same players, poll for new players or stop playing completely.
+
+        :param min_players: the minimum number of players who need to join in order to continue.
+            optional, defaults to 1
+        """
+        ipl = InteractionPipeline(
+            ButtonInteraction(
+                "checkmark",
+                helpstring=f"Press {EMOJI_FORWARD['checkmark']} to vote to continue with the same players.",
+            ),
+            ButtonInteraction(
+                "busts-in-silhouette",
+                helpstring=f"Press {EMOJI_FORWARD['busts-in-silhouette']} to vote to continue with new players.",
+            ),
```

Comment:
Do we want to support continuing with *new* players? Surely that's just starting a new game?

Reply:
Not necessarily. In ECards, new players will get a new hand of cards, but existing players will keep the same hand. This allows for players to join late to a game of ECards, or leave halfway through without ruining the game for everyone else.

Reply:
Alright, but maybe do this in a later commit. At the moment, a lot of the state is dependent on `self.players` not really changing in the games.